includes/pull-ui.php

```diff
@@ -624,3 +588,63 @@ function output_pull_errors() {
 
 	<?php
 }
+
+/**
+ * Get connections for pull
+ */
+function get_connections() {
+	$connections = array();
+
+	if ( ! empty( \Distributor\Connections::factory()->get_registered()['networkblog'] ) ) {
+		$sites = \Distributor\InternalConnections\NetworkSiteConnection::get_available_authorized_sites( 'pull' );
+
+		foreach ( $sites as $site_array ) {
+			$internal_connection = new \Distributor\InternalConnections\NetworkSiteConnection( $site_array['site'] );
+
+			$connections[] = [
+				'id'   => $internal_connection->site->blog_id,
+				'name' => untrailingslashit( $internal_connection->site->domain . $internal_connection->site->path ),
+				'url'  => untrailingslashit( preg_replace( '#(https?:\/\/|www\.)#i', '', get_site_url( $internal_connection->site->blog_id ) ) ),
```

Comment:
I'm a little unclear why name and URL are using the same (or at least similar ) values. `site->domain . site->path` ought to resolved to `get_site_url()`.

Although we don't display the name, it may be helpful to allow people to search by the name, so a search for `Make WordPress Core` would return `make.wordpress.org/core`